posthog/api/annotation.py

```diff
@@ -80,7 +80,7 @@ def _filter_request(self, request: request.Request, queryset: QuerySet) -> Query
                 queryset = queryset.filter(scope=request.GET["scope"])
             elif key == "apply_all":
                 queryset_method = queryset.exclude if str_to_bool(request.GET["apply_all"]) else queryset.filter
-                queryset = queryset_method(scope="dashboard_item")
+                queryset = queryset_method(scope=Annotation.Scope.DASHBOARD_ITEM)
```

Comment:
are we stuck with this last reference to dashboard item to avoid database faff?

Reply:
Hm, we _are_ kind of stuck database side, but actually no reason to be stuck with it in most of the code – renamed the enum variant's name